lib/scsbxml_fetcher.rb

```diff
@@ -41,11 +41,19 @@ def translate_to_scsb_xml
             },
             headers: { 'Authorization' => "Bearer #{@oauth_token}" }
           ).body
+
```

Comment:
I thought that "bad" / not found responses are valid JSON so `response_body.empty?` would return true.
Right?

We _could_ do something like:

```ruby
response = HTTParty.get("....")

if response.code >= 400 
  # add to errors and log
else
  results[barcode] = response.body
end
```